src/setup/templates/rgw.logrotate.tmpl

```diff
@@ -0,0 +1,9 @@
+TMP_LOG_PATH/*.log
+{
+    rotate 7
```

Comment:
lets increase number of files to retain as 15 atleast, since we have reduced file size.

what is the default maxsize btw, if its not specific here

Reply:
There is no default maxsize,
when maxsize is used, log files are rotated when they reach the size limit even before the specified time interval.
so size takes precedence.
using only size, it will rotate log files at the next cronjob call only.

Reply:
Keeping the rotate count = 15 for now.